setup.py

```diff
@@ -113,6 +113,7 @@ def get_git_sha() -> str:
         "PyJWT>=2.4.0, <3.0",
         "redis",
         "selenium>=3.141.0",
+        "sshtunnel>=0.4.0",
```

Comment:
Should we put an upper limit to the version range here?

Reply:
bump

Reply:
Should be fixed as part of: https://github.com/apache/superset/pull/22306